contracts/validators/request/LinkedMultiQueryValidator.sol

```diff
@@ -82,6 +82,19 @@ contract LinkedMultiQueryValidator is Ownable2StepUpgradeable, IRequestValidator
         $._supportedCircuits[CIRCUIT_ID] = IGroth16Verifier(_groth16VerifierContractAddr);
         $._supportedCircuitIds.push(CIRCUIT_ID);
 
+        _setInputToIndex("linkID", 0);
+        _setInputToIndex("merklized", 1);
+        for (uint256 i = 0; i < QUERIES_COUNT; i++) {
+            _setInputToIndex(
+                string(abi.encodePacked("operatorOutput_", Strings.toString(i))),
```

Comment:
In the `_getResponseFields` in this contract the response field of `operatorOutput_` should have underscore as we've changed it here.